cgi/CRMS.pm

```diff
@@ -1458,7 +1462,7 @@ sub LoadQueueForProject {
     $sql = 'SELECT c.id FROM candidates c'.
            ' INNER JOIN bibdata b ON c.id=b.id'.
            ' WHERE c.project=? AND b.sysid=?'.
-           ' ORDER BY c.time ASC';
```

Comment:
`c.time ASC`? `c.time DESC`? The second query is for a subset of the first and the order doesn't really matter so we'll just reuse the `ORDER BY` from the outer query.

Reply:
If the order "doesn't really matter" here, why order at all?

Reply:
Good point. Removed from second query.